app/src/main/java/com/itsaky/androidide/handlers/FileTreeActionHandler.kt

```diff
@@ -336,6 +356,7 @@ class FileTreeActionHandler : BaseEventHandler() {
       if (newFile.exists()) {
         app.toast(string.msg_file_exists, ERROR)
       } else {
+        if (autoLayout) createAutoLayout(context, directory, name)
```

Comment:
If the layout file creation fails for any reason (let's say the layout file already exists), then the process should probably fail. Also, if the layout file exists, you show an error message to the user, but still the Java file creation process continues. This shouldn't happen.

If the user has selected to create the layout file, and the layout file exists, the process should simply fail.